internal/provider/resource_image_repo_test.go

```diff
@@ -27,12 +28,14 @@ func TestImageRepo(t *testing.T) {
 		parentID: parentID,
 		name:     name,
 		bundles:  `["a", "b", "c"]`,
+		readme:   "# hello",
```

Comment:
Since readme is optional, can you add tests for adding a readme when there wasn't one, and removing a readme when there was?